packages/react-components/src/components/chart/utils/getInfo.ts

```diff
@@ -1,9 +1,9 @@
 import { DurationViewport, Viewport } from '@iot-app-kit/core/src';
```

Comment:
is this whole file doing interpolation calculations? if so could we rename it to something less generic... getInfo doesn't tell us much about what it does

Reply:
Sure thing, any suggestions ? , it is all the auxiliary functions we have for charts / TCs...

Reply:
we could split each out into a separate file in the utils folder, or move the TC/interpolation helpers closer to where they're used